src/main/java/seedu/address/model/student/AcademicYear.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.model.student;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Student's name in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidAcademicYear(String)}
+ */
+public class AcademicYear {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "The Academic Year should only contain a number between 1 to 5.";
```

Comment:
Isn't this supposed to be 1 to 6?

Reply:
I checked the google docs and it said 1 to 5? hmm can someone confirm

Reply:
@yejiadong @kerkpy